src/scenes/NftDetailPage/NftAdditionalDetails.tsx

```diff
@@ -62,14 +62,21 @@ function NftAdditionalDetails({ nft }: Props) {
           <BodyRegular color={colors.gray50}>Token ID</BodyRegular>
           <BodyRegular>{hexHandler(nft.opensea_token_id)}</BodyRegular>
           <Spacer height={16} />
-          {hasContractAddress && (
-            <StyledLinkContainer>
-              <StyledLink href={getOpenseaExternalUrl(nft)} target="_blank" rel="noreferrer">
-                <ActionText color={colors.gray50}>View on OpenSea</ActionText>
+          <StyledLinkContainer>
```

Comment:
`StyledLinkContainer` does not take up any space/have any visual appearance if it is blank so this is acceptable even if the asset is missing a contract address and an external URL